### PR TITLE
Added Poker Code

### PR DESCRIPTION
**Pull request by shim1998**

### CODE REVIEW

**Comment by CaioCordeiro** [comment]
What a great code :smile: !
Just a little tip, its good to always put a description on your PRs to make more understandable.
Thank you for contributing! :rocket:
I just have a few remarks about your PR.
I would separate this PR into 2 PRs, one for the 'Poker Code' and other to the Cardiogram

File: Poker/poker.cpp [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+    // swap(v1,v2);
+    // rep(i,0,14){
+    //     cout<<v1[i]<<' ';
+    // }
+    // cout<<'\n';
+    // rep(i,0,14){
+    //     cout<<v2[i]<<' ';
+    // }
+    // cout<<'\n';
+    // cout<<a<<' '<<b<<'\n';
```
Comment (CaioCordeiro) [inline Poker/poker.cpp:208]: I would remove this comment :smile:

File: Poker/poker.cpp [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+            }
+        }
+    }
+}
+
+int main(){ 
+    // #ifndef ONLINE_JUDGE  
+        // freopen("input.txt", "r", stdin); 
+        // freopen("output.txt", "w", stdout); 
+    // #endif
```
Comment (CaioCordeiro) [inline Poker/poker.cpp:274]: I would remove this comment smile :smile:

**Review by CaioCordeiro — CHANGES_REQUESTED** [review]

File: Poker/poker.cpp (hunk 1 above)
Comment (shim1998) [reply Poker/poker.cpp:208]: Sure :D

**Pull request closed without merging**